### DISCUSSION TITLE
No response after clicking the Login to GitHub button for PhpStorm Co-pilot plugin

### DISCUSSION THREAD

**Discussion by sennett-lau** [community/community#46433] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I've installed the GitHub Co-pilot plugin on Phpstorm, and wanted to enable the co-pilot by Login to GitHub; however, nothing is showing up after clicking the button... have tried to reinstall the plugin, restart the IDE and even the mac itself.

PhpStorm: 2022.3.2

MacOS: Ventura 13.2

GitHub Co-pilot: 1.2.1.2347
![image](https://user-images.githubusercontent.com/47737641/216831121-cbb98a2b-fdc6-4137-af0a-c559a00c84bf.png)

**Comment by chris-c-thomas** [comment, marked as answer]
So I just tested on the exact same setup and was able to successfully activate the plugin:

<img width="449" alt="Screenshot 2023-02-05 at 11 21 09 AM" src="https://user-images.githubusercontent.com/22353987/216834242-883b2668-c14b-4e9f-b6ff-d55fc545bb0e.png">

It'll generate an 8 character code. Once you copy that you'll have it open the authorization web page on GitHub in your browser that you paste it into. Is it possible the code popup is hidden behind another window or the web page is opening somewhere you're not seeing? My thought process would be that it's just waiting for you to enter that and won't move forward until you do.

My same setup:
macOS 13.2
PhpStorm 2022.3.2
Copilot 1.2.1.2347

**Reply by sennett-lau** [reply]
Somehow it works now for an unknown reason. Thanks btw

**Reply by chris-c-thomas** [reply]
No problem, glad to hear it.
